lms/templates/header/user_dropdown.html

```diff
@@ -25,36 +26,52 @@
 %>
 
 <div class="nav-item hidden-mobile">
-    <a href="${reverse('dashboard')}" class="menu-title">
+    <button class="menu-title"  aria-expanded="false" tabindex="0" aria-controls="user-menu">
         <img data-hj-suppress class="user-image-frame" src="${profile_image_url}" alt="">
         <span class="sr-only">${_("Dashboard for:")}</span>
         <span data-hj-suppress class="username">${displayname}</span>
-    </a>
+        <span class="fa fa-caret-down" aria-hidden="true"></span>
+    </button>
 </div>
 <div class="nav-item hidden-mobile nav-item-dropdown" tabindex="-1">
-    <div class="toggle-user-dropdown" role="button" aria-label=${_("Options Menu")} aria-expanded="false" tabindex="0" aria-controls="user-menu">
-        <span class="fa fa-caret-down" aria-hidden="true"></span>
-    </div>
     <div class="dropdown-user-menu hidden" aria-label=${_("More Options")} role="menu" id="user-menu" tabindex="-1">
-        % if show_gamification:
-            <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('gamma_dashboard:gamma-dashboard')}" role="menuitem">${_("Performance")}</a></div>
-            <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('gamma_dashboard:gamma-leaderboard')}" role="menuitem">${_("Leaderboard")}</a></div>
+        <div class="mobile-nav-item dropdown-item dropdown-nav-item with-border">
+            <span class="dropdown-section-name">${_("Dashboard")}</span>
+        </div>
+        <div class="mobile-nav-item dropdown-item dropdown-nav-item">
+            <a href="${reverse('dashboard')}" role="menuitem">${_("My courses")}</a>
+        </div>
+        % if show_program_listing:
+            <div class="mobile-nav-item dropdown-item dropdown-nav-item">
+                <a href="${reverse('program_listing_view')}" role="menuitem">${_("My programs")}</a>
+            </div>
         % endif
+        <div class="mobile-nav-item dropdown-item dropdown-nav-item with-border">
+            <span class="dropdown-section-name">${_("Manage information")}</span>
+        </div>
         <div class="mobile-nav-item dropdown-item dropdown-nav-item">
-            <a href="${reverse('learner_profile', kwargs={'username': username})}" role="menuitem">${_("Profile")}</a>
+            <a href="${reverse('account_settings')}" role="menuitem">${_("My account")}</a>
         </div>
         <div class="mobile-nav-item dropdown-item dropdown-nav-item">
-            <a href="${reverse('account_settings')}" role="menuitem">${_("Account")}</a>
+            <a href="${reverse('learner_profile', kwargs={'username': username})}" role="menuitem">${_("My profile")}</a>
+        </div>
+        <div class="mobile-nav-item dropdown-item dropdown-nav-item">
+            <a href="https://edx.readthedocs.io/projects/open-edx-learner-guide/en/open-release-quince.master/SFD_dashboard_profile_SectionHead.html" rel="menuitem" target="_blank">${_("Help Center")}</a>
```

Comment:
Are we sure that it's a good way to set links like this one? Maybe it should be taken from the config?

Reply:
Yeah, let's follow the common pattern and set the link through the settings